core/metamodel/src/main/java/org/apache/isis/core/metamodel/adapter/ObjectAdapterProvider.java

```diff
@@ -71,6 +86,11 @@ ObjectAdapter adapterForViewModel(
         @Programmatic
         ObjectAdapterProvider getObjectAdapterProvider();
         
+//        @Programmatic
+//        default Oid oidFor(Object domainObject) {
+//            return getObjectAdapterProvider().oidFor(domainObject);
+//        }
+        
```

Comment:
seemed a nice addition, but did not work at my first attempt of implementing it; commented out for now

Reply:
ok